HARK/ConsumptionSaving/ConsIndShockModel.py

```diff
@@ -1609,7 +1611,7 @@ def __init__(self, verbose=1, quiet=False, **kwds):
         self.quiet = quiet
         self.solve_one_period = make_one_period_oo_solver(ConsPerfForesightSolver)
         set_verbosity_level((4 - verbose) * 10)
-
+        self.bilt = {}
```

Comment:
I believe that in the "HARK 2.0 alpha" PR, CDC's use of mispelt English words was to make it easier to find-and-replace them all with a more readable, less opaque name.

Is this being used for anything besides conditions? I would be happy to name this `conditions`. Or could be `built`. Or `derived`.